src/main/java/com/onlinebookstore/dto/user/UserResponseDto.java

```diff
@@ -0,0 +1,9 @@
+package com.onlinebookstore.dto.user;
+
+import lombok.Data;
+
+@Data
+public class UserResponseDto {
+    private Long id;
+    private String email;
```

Comment:
Let’s  provide other fields (exclude password)

Reply:
Comment not fixed
Example of response body:

```
{
  "id": 1,
  "email": "john.doe@example.com",
  "firstName": "John",
  "lastName": "Doe",
  "shippingAddress": "123 Main St, City, Country"
}
```